pod2daemon/Makefile

```diff
@@ -5,7 +5,8 @@ PACKAGE_NAME = github.com/projectcalico/calico/pod2daemon
 # Name of the images.
 # e.g., <registry>/<name>:<tag>
 FLEXVOL_IMAGE  ?=pod2daemon-flexvol
```

Comment:
The plan is to remove this later. I think we should build this image until we have fully removed the flexvolume from everything (operator, EE, etc.) since this is just the first part of that transition.